spec/unit/libraries/conf_helpers_spec.rb

```diff
@@ -108,26 +106,44 @@
         }
       }
     end
-    let(:conf_file) { double('Pathname.new(conf_path)') }
 
-    it 'should return the config in a hash' do
-      expect(Pathname).to receive(:new).with(conf_path).and_return(conf_file)
-      expect(conf_file).to receive(:exist?).and_return(true)
-      expect(conf_file).to receive(:readlines).and_return(IO.readlines('spec/reference/read_test.conf'))
+    it 'should call parse_config' do
```

Comment:
I am not sure this test name is very descriptive of what we are actually testing since I don't see anything about parse in the test itself.